core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXV1Controller.java

```diff
@@ -85,15 +85,17 @@
 
     @Override
     public StackViewV4Responses list(String environmentName, String environmentCrn) {
-        StackViewV4Responses stackViewV4Responses;
-        if (!Strings.isNullOrEmpty(environmentName)) {
-            stackViewV4Responses =  stackOperations.listByEnvironmentName(
+        if (StringUtils.isNotEmpty(environmentName) && StringUtils.isNotEmpty(environmentCrn)) {
```

Comment:
what if environmentCrn is null or empty? you need to fix this case.

Reply:
hahahahahaha I don't need to, this is an expected case, the repositories are prepared for this. I was shocked as well.